jekyll/_cci2/faq.md

```diff
@@ -1,41 +1,69 @@
 ---
 layout: classic-docs
-title: "Frequently Asked Questions (FAQ)"
-short-title: "FAQ"
-description: "Frequently asked questions about CircleCI 2.0"
-categories: [getting-started]
-order: 50
+title: "Migration FAQ"
+short-title: "Migration FAQ"
+description: "Frequently asked questions about CircleCI 2.0 Migration"
+categories: [migration]
+order: 1
 ---
 
+* Contents
+{:toc}
+
+## Why Migrate from CircleCI 1.0 to 2.0?
+
+- No more queued jobs! CircleCI 2.0 includes a significant rewrite to resource utilization that eliminates waiting for the last Job to use available containers by breaking Jobs into Steps. 
```

Comment:
thanks, how about 'CircleCI 2.0 includes a significant rewrite of container utilization to run more jobs faster and to prevent available containers from sitting idle.'